controllers/nodehealthcheck_controller.go

```diff
@@ -474,6 +475,7 @@ func (r *NodeHealthCheckReconciler) remediate(node *v1.Node, nhc *remediationv1a
 
 	// always update status, in case patching it failed during last reconcile
 	resources.UpdateStatusRemediationStarted(node, nhc, remediationCR)
+	metrics.ObserveNodeHealthCheckRemediationCreated(node.GetName(), remediationCR.GetNamespace(), remediationCR.GetKind())
```

Comment:
can you please add a comment why this isn't inside the `if created{}` block below, for future us wondering about it? 😎 Something like `// ensure to provide correct metrics in case the CR existed already after a pod restart`?

Reply:
Done

Reply:
thanks!